setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name="mplc",
-    version='0.0.0.1',
+    version='0.1',
     author="SubstraFoundation",
     author_email="contact@substra.org",
     description="A distributed learning contributivity package",
```

Comment:
What about:
```suggestion
    description="A distributed learning contributivity measure package",
```
?

Reply:
Yeah the description needs to be changed I approve